src/geometry/interaction_groups.rs

```diff
@@ -80,70 +83,70 @@ bitflags::bitflags! {
     #[cfg_attr(feature = "serde-serialize", derive(Serialize, Deserialize))]
     #[derive(Copy, Clone, PartialEq, Eq, Debug, Hash)]
     pub struct Group: u32 {
+        /// The default group.
+        const DEFAULT = 1 << 0;
         /// The group n°1.
-        const GROUP_1 = 1 << 0;
```

Comment:
Not sure I agree on shifting those groups though:
- it makes a tedious migrations for users
- javascript bindings will have to adapt (not hard, but tedious/error prone again)

the naming becomes a bit inconsistent, I would prefer `DEFAULT` and `GROUP_1` being the same, or straight up removing DEFAULT, and add that's the default as a comment.

Reply:
Ya I agree